src/esp/nav/PathFinder.cpp

```diff
@@ -951,13 +958,13 @@ bool PathFinder::Impl::build(const NavMeshSettings& bs,
   const int numVerts = mesh.vbo.size();
   const int numIndices = mesh.ibo.size();
   const float mf = std::numeric_limits<float>::max();
-  vec3f bmin(mf, mf, mf);
-  vec3f bmax(-mf, -mf, -mf);
+  Mn::Vector3 bmin(mf, mf, mf);
+  Mn::Vector3 bmax(-mf, -mf, -mf);
 
   for (int i = 0; i < numVerts; ++i) {
-    const vec3f& p = mesh.vbo[i];
-    bmin = bmin.cwiseMin(p);
-    bmax = bmax.cwiseMax(p);
+    const Mn::Vector3& p = mesh.vbo[i];
+    bmin = Mn::Math::min(bmin, p);
+    bmax = Mn::Math::max(bmax, p);
   }
```

Comment:
Was working on something unrelated and came across [MeshTools::boundingRange()](https://doc.magnum.graphics/magnum/namespaceMagnum_1_1MeshTools.html#a2516ccb77fe746d8461da48eaf7e9ef3), so feel free to use it here :)